docs/src/add-services/varnish.md

```diff
@@ -56,20 +56,20 @@ The VCL file you provide has three specific requirements over and above the VCL
 
 The absolute bare minimum VCL file is:
 
-```bash
+```bash {location=".platform/config.vcl"}
 sub vcl_recv {
-    set req.backend_hint = application.backend();
+    set req.backend_hint = varnishstats.backend();
 }
 ```
 
-Where `application` is the name of the relationship defined in `services.yaml`.
-(If the relationship was named differently, use that name instead.)
+Where `varnishstats` is the name of the relationship defined in `services.yaml`.
+(If the relationship was named differently in `<RELATIONSHIP_NAME>`, use that name instead.)
```

Comment:
On further investigation, I think this change is not correct. The relationship here should be the one Varnish has to the app, which isn't defined as `varnishstats`. The `varnishstats` relationship only shows stats and doesn't do the things in this file.

It might be easier when the stats relationship is gone as part of #2352, so I might just suggest closing this one